### ISSUE TITLE
When used in pure js project, eslint reports error.

### ISSUE THREAD

**Issue by gavinliu6** [schoero/eslint-plugin-better-tailwindcss#194]
I have a pure js project (open with vscode), after I used this plugin, eslint reported error:

Couldn't find tsconfig.json
……
An unexpected error occurred:
TypeError: matchPath is not a function

My eslint config:

```js
import stylistic from '@stylistic/eslint-plugin'
import eslintPluginBetterTailwindcss from 'eslint-plugin-better-tailwindcss'
import simpleImportSort from 'eslint-plugin-simple-import-sort'
import eslintPluginVue from 'eslint-plugin-vue'

export default [
  stylistic.configs.recommended,
  ...eslintPluginVue.configs['flat/recommended'],
  {
    plugins: {
      'simple-import-sort': simpleImportSort,
      'better-tailwindcss': eslintPluginBetterTailwindcss,
    },
    rules: {
      ...eslintPluginBetterTailwindcss.configs['recommended-warn'].rules,
      'simple-import-sort/imports': 'error',
      'simple-import-sort/exports': 'error',
      '@stylistic/brace-style': ['error', '1tbs', { allowSingleLine: true }],
      '@stylistic/comma-dangle': ['error', {
        arrays: 'always-multiline',
        objects: 'always-multiline',
        functions: 'never',
      }],
      'vue/multi-word-component-names': 'off',
      'vue/html-self-closing': 'off',
      'better-tailwindcss/no-unregistered-classes': [
        'error',
        {
          detectComponentClasses: true,
          ignore: ['Layout'],
        },
      ],
    },
    settings: {
      'better-tailwindcss': {
        entryPoint: '.vitepress/theme/style.css',
      },
    },
  },
]
```
